### DISCUSSION TITLE
How to organize pairs of subgraphs in social networks into batch data for training?

### DISCUSSION THREAD

**Discussion by finquick** [pyg-team/pytorch_geometric#6158] [Q&A]
In the social network, I regard each node as a person and extract the first-order neighbor into a subgraph, the source network and the target network are subgraph extraction, and then compare the similarity of the two subgraphs, the current practice is not organized into the batch, can not use PYG accelerated training characteristics, is there a reference to imitate the demo, thank you!

**Comment by wsad1** [comment]
For pairs of graphs you need to modify the  `__inc__`  in `dataloader` as described [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/batching.html#pairs-of-graphs)
